CONTRIBUTING.md

```diff
@@ -264,6 +273,55 @@ Finally, when a point release is ready to be made:
 Note, although we aim to support only a single release at a time, the process stated above could be
 used for multiple previous versions.
 
+## Stable Release Updates (SRU)
```

Comment:
```suggestion
## Stable Release Updates
```

Reply:
Do you want me to rename this as well @alexanderbez @marbar3778? All in all, I don't think this naming is so bad, this should help clarify what backports are: updates to stable releases (aka maintenance releases). Wdyt?

Reply:
I think we should stick with point releases.

Reply:
Done, thanks